targets/TARGET_STM/TARGET_STM32F0/serial_device.c

```diff
@@ -92,43 +101,43 @@ static void uart3_8_irq(void)
 #if defined(TARGET_STM32F091RC)
```

Comment:
I was wondering if the target necessary defines were still necessary? Haven't looked at the specific processor yet.

Reply:
yes it is required because the IRQ vector name is different and also the __HAL_GET_PENDING_IT macro is available only on F09x devices